### DISCUSSION TITLE
Updateable list of deps based on GitHub repos?

### DISCUSSION THREAD

**Discussion by strophy** [renovatebot/renovate#19132] [Request Help] [answered]
Hi, I am co-maintainer of [rvolosatovs/docker-protobuf](https://github.com/rvolosatovs/docker-protobuf), which serves as a handy build tool for devs using protobuf. It requires frequent updating through [PRs like this one](https://github.com/rvolosatovs/docker-protobuf/pull/112/files), and the other maintainer has written [a short Go program](https://github.com/rvolosatovs/docker-protobuf/blob/main/check-versions.go) to check for version updates. Actually doing the update and opening the PR is still a manual task for us.

It seems to me that this is a problem that can be solved by some basic text-file oriented list of package sources to accompany the list of versions we already maintain, i.e. dependency management. Since the Dockerfile downloads each of these packages during the build anyway, the source URLs could maybe even be read in from this dependency management format directly?  I'm not very familiar with package management, but I wonder what the community here would recommend for a format that can be readily handled by Renovate (and/or Dependabot) to check GitHub repos for updated versions and automate the frequent version update PRs we need to make.

Thanks in advance for ideas!

**Comment by viceice** [comment, marked as answer]
you should be able to use the regex manager to update those versions.

**Reply by strophy** [reply]
This looks perfect, thank you!
